dask/array/ghost.py

```diff
@@ -27,78 +15,20 @@ def fractional_slice(task, axes):
     >>> fractional_slice(('x', 2.9, 5.1), {0: 2, 1: 3})  # doctest: +SKIP
     (getitem, ('x', 3, 5), (slice(0, 2), slice(-3, None)))
     """
-    rounded = (task[0],) + tuple(int(round(i)) for i in task[1:])
-
-    index = []
-    for i, (t, r) in enumerate(zip(task[1:], rounded[1:])):
-        depth = axes.get(i, 0)
-        if t == r:
-            index.append(slice(None, None, None))
-        elif t < r:
-            index.append(slice(0, depth))
-        elif t > r and depth == 0:
-            index.append(slice(0, 0))
-        else:
-            index.append(slice(-depth, None))
 
-    index = tuple(index)
+    warn('ghost module will be renamed to overlap, '
+         'use overlap.fractional_slice.',
+         DeprecationWarning)
```

Comment:
`DeprecationWarning`s are ignored by default on non-python 2, so users won't see this warning (which is unhelpful). In other parts of dask we have been using raw warnings prefixed with the string `"DeprecationWarning:"`, which seems to work fairly well.

Reply:
Good point, completely forgot those are ignored with default warnings. I've updated the warning type and message text.